ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -4080,8 +4097,8 @@ public List<Partition> dropPartitions(String dbName, String tableName,
           .map(ss -> ss.getTxnMgr().getCurrentTxnId()).orElse(0L);
         dropOptions.setTxnId(txnId);
       }
-      List<org.apache.hadoop.hive.metastore.api.Partition> partitions = getMSC().dropPartitions(dbName, tableName,
-          partitionExpressions, dropOptions);
+      List<org.apache.hadoop.hive.metastore.api.Partition> partitions = getMSC().dropPartitions(
+          tableName, partsSpec, dropOptions, null);
```

Comment:
can we have a default method in API to avoid passing nulls?

Reply:
Do we need a new API for it? There seems already so many APIs for `dropPartitions`.

Reply:
not sure, however, passing NULLs isn't great either